countries/ch.json

```diff
@@ -134,11 +134,5 @@
         "url": "https://framagenda.org/remote.php/dav/public-calendars/KwNwGA232xD38CnN/?export",
         "type": "api",
         "id": 801
-    },
-    {
-        "name": "Fresque des Déchets",
-        "url": "https://calendar.google.com/calendar/ical/greendonut.info%40gmail.com/public/basic.ics",
-        "type": "api",
-        "id": 802
```

Comment:
Instead of removing this data source, this id could simply be updated to `15`? This ICS source in ch.json is not clashing with the one in fr.json, so it is reasonable to reuse the same id with different sources.